cmd/snap/cmd_version.go

```diff
@@ -58,7 +58,11 @@ func printVersions(cli *client.Client) error {
 
 	fmt.Fprintf(w, "snap\t%s\n", cmd.Version)
 	fmt.Fprintf(w, "snapd\t%s\n", sv.Version)
-	fmt.Fprintf(w, "series\t%s\n", sv.Series)
+	if sv.Architecture != "" {
```

Comment:
Why not
```
if sv.Architecture != "" {
    fmt.Fprintf(w, "architecture\t%s\n", sv.Architecture)
} else {
    fmt.Fprintf(w, "architecture\t-\n")
}
```

BTW. we can't use `arch` as Arch uses `ID=arch` in /etc/os-release and we print that out already:
```
snap    2.41-1
snapd   +git443.g7abba46
series  16
arch    -
kernel  5.2.11-arch1-1-ARCH
```